packages/barrier/SOURCES/barrier.sgifixes.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 18b242db..b7694414 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -18,9 +18,9 @@
+ cmake_minimum_required (VERSION 3.4)
+ project (barrier C CXX)
+ 
+-option (BARRIER_BUILD_GUI "Build the GUI" ON)
++option (BARRIER_BUILD_GUI "Build the GUI" OFF)
+ option (BARRIER_BUILD_INSTALLER "Build the installer" ON)
+-option (BARRIER_BUILD_TESTS "Build the tests" ON)
++option (BARRIER_BUILD_TESTS "Build the tests" OFF)
+ option (BARRIER_USE_EXTERNAL_GTEST "Use external installation of Google Test framework" OFF)
+ 
+ set (CMAKE_EXPORT_COMPILE_COMMANDS ON)
+@@ -342,7 +342,7 @@ elseif (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+         )
+     endif()
+ 
+-elseif (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
++elseif (${CMAKE_SYSTEM_NAME} MATCHES "Linux|IRIX64")
```

Comment:
IRIX64 works fine for larger machines - but this might fail when building on an O2 (32bit IRIX).

Perhaps use `Linux|IRIX*`?

Reply:
That should work great, I've tested a build with that and seems good.

Reply:
Should be resolved now